docs/JLBP-0006.md

```diff
@@ -82,18 +82,18 @@ Given this scenario, here are the possible combinations of renamings:
     will only choose one of them to use. Users are forced to change their code
     between versions, but only for library surface that was changed.
 
-  - **Case 2: Rename Maven ID**. Users can easily pull in both the old jar
-    (`g1:a1:1.0.0`) and the new jar (`g2:a2:2.0.0`) into their classpath
-    accidentally through transitive dependencies because Maven artifact
+  - **Case 2: Rename Maven ID**. Build systems can unexpectedly pull in the JAR files of both the
+    old artifact (`g1:a1:1.0.0`) and the new artifact (`g2:a2:2.0.0`) into their class path
+    through transitive dependencies, because Maven artifact
     resolution treats the two artifacts as distinct.  This results in
     "overlapping classes" since they have classes with the same fully-qualified
     path. This can lead to runtime exceptions such as `NoSuchMethodDefError`
     and can be a compile-time error in Java 9 and later. **NEVER DO THIS**.
 
 - **Rename Java package**:
   - **Case 3: Keep Maven ID**. The classes from `g1:a1:1.0.0` and
-    `g1:a1:2.0.0` could technically be used together, but since they share the
-    Maven group ID and artifact ID, only one jar can be pulled in.
+    `g1:a1:2.0.0` could be used together, but since the two artifacts share the
+    Maven group ID and artifact ID, the build system selects only one of them.
```

Comment:
"selects only one of them" --> adds only one of them to the class path.

Reply:
Updated.